Update to Infinite cage S120 (#1244)
pi-base/data: spaces/S000120/properties/P000206.md

---
space: S000120
property: P000206
value: true
---

Observe that every point except $(0,n+\frac{1}{2},0)$ (end of some segment $B_k$)
admits a neighbourhood closed in $\mathbb R^3$ (i.e. {P55},

> which is hereditary on open subsets).

Author: We take $\mathbb R^3$-closed nbhd which is automatically complete and from this we infer that the open nbhd (the interior) is P55.

> Therefore Player 1 cannnot pick $x_n$ of this form,
> becuse the next move of Player 2 can reduce the game

Reviewer: ```suggestion
Therefore Player 1 cannot pick $x_n$ of this form,
because the next move of Player 2 can reduce the game
```

---
space: S000120
property: P000206
value: true
---

> Observe that every point except $(0,n+\frac{1}{2},0)$ (end of some segment $B_k$)
> admits a neighbourhood that is closed in $\mathbb R^3$ (i.e. {P55}).
> Its interior in $X$ is then {P55} as well.
> 
> Therefore Player 1 cannot pick $x_n$ of this form,
> because the next move of Player 2 can reduce the game
> to the case of a completely metrizable space $V_n$ which is {P206}
> (let us call it the "C-scenario").
> 
> On the other hand, Player 2 can always make sure that precisely one $B_k$-end lies in
> $V_0$, therfore forcing Player 1 to either loose by picking a constant sequence
> $\{x_n\}_{n\geq 1}$ or by triggering the C-scenario.

Reviewer 2: ```suggestion
Observe that every point of $X$ not of the form $(0,n+\frac{1}{2},0)$ (endpoint of some segment $B_k$)
admits a neighbourhood in $X$ that is closed in $\mathbb R^3$ (hence {P55}).
Its interior in $X$ is then {P55} as well.

Therefore Player 1 cannot pick $x_n$ of this form,
because the next move of Player 2 would reduce the game
to the case of a completely metrizable space $V_n$, which is {P206}
[(Explore)](https://topology.pi-base.org/spaces?q=Completely+metrizable%2B%7EEmpty%2B%7EStrongly+Choquet).
(Let us call this the "C-scenario".)

On the other hand, Player 2 can always make sure that precisely one $B_k$-endpoint lies in
$V_0$, therefore forcing Player 1 to either lose by picking a constant sequence
$\{x_n\}_{n\geq 1}$ or by triggering the C-scenario.
```
